src/App/App.js

```diff
@@ -41,11 +43,35 @@ class App extends Component {
      })
   }
 
-  // todo ==> hook up delete button on MovieProfile -- use redirect in router
   deleteMovie = (id) => {
     this.setState({ allMovies: this.state.allMovies.filter(movie => movie.id !== id)} )
   }
 
+  findMovie = (event, searchTerm) => {
+    event.preventDefault()
+    console.log(searchTerm)
+    this.setState({ foundMovies: this.allMovies.find(movie => movie.title.includes(searchTerm))} )
+    console.log(this.state.foundMovies)
+  }
+
```

Comment:
This function is setting state for found movies based on the search term passed from the SearchBar component input field.